migrate/db-seed-commonjs-pkg/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -eux
+
+yarn prisma db pull --print
```

Comment:
confirmation that `db push` worked?

Reply:
It's a copy paste from your script, so probably